src/Application/Filters/Actions/ActionWatershedFilter.cc

```diff
@@ -159,9 +158,16 @@ class WatershedFilterAlgo : public ITKFilter
 	double watershedLevel_val_;
 
 	std::vector<std::string> dst_layer_ids_;
-	std::vector<_LabelType> watershed_image_output_labels_;
-  
+	std::vector<WtrLabelType> watershed_image_output_labels_;
+
 public:
+
+	void report_debug_message( const std::string& msg )
```

Comment:
Yes, please do. CORE_LOG_DEBUG is sufficient.

> On Jun 30, 2016, at 11:02 AM, Braxton <notifications@github.com> wrote:
> 
> In src/Application/Filters/Actions/ActionWatershedFilter.cc <https://github.com/SCIInstitute/Seg3D/pull/161#discussion_r69170336>:
> 
> >  public:
> > +
> > +	void report_debug_message( const std::string& msg )
> I was following the same methodology as context->report_error( msg ); I can take it out if you would like.
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub <https://github.com/SCIInstitute/Seg3D/pull/161/files/ac9d5b2fba17d4584d106c0a5fbaa0287b7b94c8..c5d884cc49746da3b8976489652bdf940ea0af76#r69170336>, or mute the thread <https://github.com/notifications/unsubscribe/ABVaUWxFvPU7_ax0gh4EJNttumSGDaOdks5qQ_aVgaJpZM4I8ehi>.
>